Doc/library/binascii.rst

```diff
@@ -49,7 +49,7 @@ The :mod:`binascii` module defines the following functions:
       Added the *backtick* parameter.
 
 
-.. function:: a2b_base64(string, strict_mode=False)
+.. function:: a2b_base64(string, *, strict_mode=False)
```

Comment:
"string" is positional only, so perhaps:

```suggestion
.. function:: a2b_base64(string, /, *, strict_mode=False)
```

A

Reply:
Good catch, thanks